README.md

```diff
@@ -1 +1,104 @@
-# angular-ui-router-readme
\ No newline at end of file
+# What is ui-router?
+
+## Overview
+
+`ngRoute` provides us with basic functionality, but the leading router for Angular is `uiRouter`. This provides much more, such as nested views, multiple views and a few different offerings. Let's take a look at how `uiRouter` does things instead.
+
+## Objectives
+
+- Describe ui-router and ui-view
+- Write a route using ui-router
+- Integrate a Controller with ui-router
+- Integrate a template with ui-router
+
+## uiRouter
+
+`uiRouter` is quite similar to `ngRoute`, but each "route" is known as a "state". We define states, not routes.
+
+Each state that we define is quite similar to a route in `ngRoute`, but we have to give each state a name too. This allows us to have children states inside of a state.
+
+Let's take a look at how we'd define a route in `ngRoute`:
+
+```js
+angular
+	.module('app', ['ngRoute'])
+	.config(function ($routeProvider) {
+		$routeProvider
+			.when('/user', {
+				templateUrl: 'views/user.html',
+				controller: 'UserController'
+			});
+	});
+```
+
+And how we'd define a route in `uiRouter`:
+
+```js
+angular
+	.module('app', ['ui.router'])
+	.config(function ($stateProvider) {
+		$stateProvider
+			.state('user', {
+				url: '/user',
+				templateUrl: 'views/user.html',
+				controller: 'UserController'
+			});
+	});
+```
+
+You can see they're very similar. We've moved the URL into the configuration object, and replaced the URL with the state's name. We're also using `$stateProvider` instead of `$routeProvider`.
+
+Instead of using `ng-view`, we use `ui-view` to place the states in our HTML.
+
+We'll go into more advanced concepts with `uiRouter` soon - things such as nested views and resolving data.
+
+## uiSref
+
```

Comment:
Present the problem of not using ui-sref here first - if we want to change what a URL is, we'll have to update all of the links manually. Then, ui-sref to the rescue!